lib/ui/shared/text_input_field.dart

```diff
@@ -0,0 +1,36 @@
+// Flutter imports:
+import 'package:flutter/material.dart';
+
+// Project imports:
+import 'package:paintroid/ui/theme/data/paintroid_theme.dart';
+
+class TextInputField extends StatelessWidget {
+  final TextEditingController controller;
+  final String? hintText;
+  final String? Function(String?)? validator;
+
+  const TextInputField({
+    super.key,
+    required this.controller,
+    this.hintText,
+    required this.validator,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return TextFormField(
+      controller: controller,
+      decoration: InputDecoration(
+        hintText: hintText ?? '',
+        hintStyle: PaintroidTheme.of(context).textTheme.bodySmall!.apply(
```

Comment:
same here with nulll assertion
Also why not make a new style if we need this more often?